static/js/actions/ui.js

```diff
@@ -128,3 +128,6 @@ export const setNavDrawerOpen = createAction(SET_NAV_DRAWER_OPEN);
 
 export const SET_LEARNER_CHIP_VISIBILITY = 'SET_LEARNER_CHIP_VISIBILITY';
 export const setLearnerChipVisibility = createAction(SET_LEARNER_CHIP_VISIBILITY);
+
+export const SET_TO_PEARSON_SITE_DIALOG_VISIBILITY = 'SET_TO_PEARSON_SITE_DIALOG_VISIBILITY';
```

Comment:
I would suggest calling the action `PEARSON_TOS_DIALOG` or similar, `TO_PEARSON_SITE_DIALOG` is just sort of a confusing name.